identity-apps-core/apps/authentication-portal/src/main/webapp/includes/language-switcher.jsp

```diff
@@ -115,8 +116,8 @@
         if (localeFromCookie) {
             return localeFromCookie;
         } else if (localeFromUrlParams) {
-            const firstLangFromUrlParams = decodeURIComponent(localeFromUrlParams).split(" ")[0];
-            return encodeURIComponent(firstLangFromUrlParams);
+            const firstLangFromUrlParams = localeFromUrlParams.split(" ")[0];
```

Comment:
There were some problems in splitting the encoded localeFromUrlParams. Please verify

Reply:
I have checked this scenario by passing two locales which are seperated by a space and the first locale is picked correctly by splitting from a space.